docs/source/technical_tutorials/visualization/visualize_lonboard.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    for item in items:
</pre><p>since we can pass the s3 urls directly into read_csv, I'd remove this custom list of objects and just extract a list of s3 urls</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/487/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>